### PR TITLE
Note to clarify difference between Uyuni and SUMA

### PR DESCRIPTION
**Pull request by dvosburg**
Uyuni - 'zypper up'
SUMA - 'zypper patch'

# Description
Clarifies between Uyuni update and SUMA patch

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

None

### CODE REVIEW

**Review by juliogonzalez — APPROVED** [review]
LGTM, but I am adding more reviewers from the doc squad, so they can tell :-)

File: modules/installation-and-upgrade/pages/server-minor-upgrade-uyuni.adoc [hunk 1]
```diff
@@ -44,6 +44,11 @@ zypper ref
 ----
 zypper up 
 ----
+[NOTE]
+====
+Uyuni is different from SUSE Manager in this step.  SUSE Manager uses 'zypper patch', but Uyuni requires 'zypper up'
+====
+
 +
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/server-minor-upgrade-uyuni.adoc:52]: This is WIP.  Proper ifdef will follow.  ATM, github does not let me a proper patch...

```suggestion
----
zypper patch
----
----
zypper up
----
[NOTE]
====
Uyuni is different from SUSE Manager in this step.
SUSE Manager uses 'zypper patch', but Uyuni requires 'zypper up'.
====

+
```

**Review by keichwa — APPROVED** [review]
This is just a proposal.  I think it would be enough if we'd add the note to uyuni documentation only.  But feel free to refuse or adjust!  Thanks, Don!

File: modules/installation-and-upgrade/pages/server-minor-upgrade-uyuni.adoc (hunk 1 above)
Comment (juliogonzalez) [reply modules/installation-and-upgrade/pages/server-minor-upgrade-uyuni.adoc:52]: I don't think ifdef is needed here. Please check the filename. Seems to be 100% specific to Uyuni.

Comment (keichwa) [reply modules/installation-and-upgrade/pages/server-minor-upgrade-uyuni.adoc:52]: Ok, the less ifdefs the better!  I adjusted my suggestion to a minimum.

**Pull request merged**